airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.yaml

```diff
@@ -0,0 +1,99 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/hubspot
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: HubSpot Source Spec
+  type: object
+  required:
+    - start_date
+    - credentials
+  additionalProperties: true
+  properties:
+    start_date:
+      type: string
+      title: Start Date
+      pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$
+      description: >-
+        UTC date and time in the format 2017-01-25T00:00:00Z. Any data before
+        this date will not be replicated.
+      examples:
+        - "2017-01-25T00:00:00Z"
+    credentials:
+      title: Authentication mechanism
+      description: Choose how to authenticate to HubSpot.
+      type: object
+      oneOf:
+        - type: object
+          title: Authenticate via HubSpot (OAuth)
+          required:
+            - client_id
+            - client_secret
+            - refresh_token
+            - credentials_title
+          properties:
+            credentials_title:
+              type: string
+              title: Credentials Title
+              description: Name of the credentials set
+              const: OAuth Credentials
```

Comment:
This is slightly different than the `spec.json` counterpart: previously `default` and `enum` were defined, but it looks like that's not necessary anymore and was causing test failures.

Reply:
Yup! Thanks for the change